src/sentry/integrations/slack/integration.py

```diff
@@ -172,11 +175,13 @@ def build_integration(self, state):
         # are using in post_install to send messages to slack
         if state.get("integration_id"):
             metadata["installation_type"] = "migrated_to_bot"
+            metadata["migrated_at"] = six.text_type(datetime.utcnow()).split(".")[0]
```

Comment:
@MeredithAnya Wondering if there is a better date method we could use that wouldn't require us doing this string manipulation